### ISSUE TITLE
Packagecloud.io install on Fedora 24 can fail under some circumstances

### ISSUE THREAD

**Issue by JonasT** [git-lfs/git-lfs#1652]
I just tried to upgrade git-lfs to the newest version.
However, I had installed the old one directly from the rpm (without the repo). Therefore, I removed it and I used the packagecloud.io script to add the repo to get the latest version & easy upgrades:
```
[root@falcon jonas]# dnf clean all
33 files removed
[root@falcon jonas]# dnf update
Fedora 24 - x86_64 - VirtualBox                 449 kB/s |  50 kB     00:00    
Fedora 24 - x86_64 - Updates                    1.5 MB/s |  18 MB     00:11    
Fedora 24 - x86_64                              1.4 MB/s |  47 MB     00:34    
Copr repo for seafile owned by pkerling          11 kB/s | 9.2 kB     00:00    
Dependencies resolved.
Nothing to do.
Complete!
[root@falcon jonas]# dnf search git-lfs
Last metadata expiration check: 0:00:12 ago on Tue Nov 15 03:20:58 2016.
Error: No matches found.
[root@falcon jonas]# curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
Detected operating system as fedora/24.
Checking for curl...
Detected curl...
Downloading repository file: https://packagecloud.io/install/repositories/github/git-lfs/config_file.repo?os=fedora&dist=24&source=script
done.
Installing pygpgme to verify GPG signatures...
Redirecting to '/usr/bin/dnf install -y pygpgme --disablerepo=github_git-lfs' (see 'man yum2dnf')

Importing GPG key 0xD59097AB:
 Userid     : "packagecloud ops (production key) <ops@packagecloud.io>"
 Fingerprint: 418A 7F2F B0E1 E6E7 EABF 6FE8 C2E7 3424 D590 97AB
 From       : https://packagecloud.io/github/git-lfs/gpgkey
github_git-lfs-source                           174  B/s | 296  B     00:01    
Error: installed package python2-pygpgme-0.3-18.fc24.x86_64 obsoletes pygpgme < 0.3-17 provided by pygpgme-0.3-15.fc24.x86_64
(try to add '--allowerasing' to command line to replace conflicting packages)
Installing yum-utils...
Redirecting to '/usr/bin/dnf install -y yum-utils --disablerepo=github_git-lfs' (see 'man yum2dnf')

Last metadata expiration check: 0:00:03 ago on Tue Nov 15 03:21:45 2016.
Package yum-utils-1.1.31-509.fc24.noarch is already installed, skipping.
Dependencies resolved.
Nothing to do.
Complete!
Generating yum cache for github_git-lfs...
Redirecting to '/usr/bin/dnf -q makecache -y --disablerepo=* --enablerepo=github_git-lfs' (see 'man yum2dnf')

Importing GPG key 0xD59097AB:
 Userid     : "packagecloud ops (production key) <ops@packagecloud.io>"
 Fingerprint: 418A 7F2F B0E1 E6E7 EABF 6FE8 C2E7 3424 D590 97AB
 From       : https://packagecloud.io/github/git-lfs/gpgkey

The repository is setup! You can now install packages.
[root@falcon jonas]# dnf search git-lfs
Last metadata expiration check: 0:00:07 ago on Tue Nov 15 03:22:03 2016.
Error: No matches found.
[root@falcon jonas]# dnf install git-lfs
Last metadata expiration check: 0:00:10 ago on Tue Nov 15 03:22:03 2016.
No package git-lfs available.
Error: Unable to find a match.
[root@falcon jonas]# whereis git-lfs
git-lfs:
[root@falcon jonas]# 
```
... however as you can see, this has gotten me nowhere. Does someone have an idea what the problem might be?

**Comment by technoweenie** [comment]
@andyneff any ideas? It looks like the packagecloud repo was installed with the correct gpg key, but git-lfs still isn't found.

**Comment by JonasT** [comment]
I already tried `dnf clean all` and deleting the repository from `/etc/yum.repos.d` and running the script again, however I end up with the same result (no git-lfs in packages).

**Comment by technoweenie** [comment]
Fixed as of #1685
